src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -112,7 +112,20 @@ const policy::DeviceParams GetDeviceParams(
   device_params.device_handle = device_handle;
   return device_params;
 }
-}
+
+struct HMILevelPredicate
+    : public std::unary_function<ApplicationSharedPtr, bool> {
+  HMILevelPredicate(const mobile_api::HMILevel::eType level) : level_(level) {}
```

Comment:
@dev-gh Looks that here should be added `explicit` keyword according SDL coding style